src/Entities/Player.h

```diff
@@ -218,6 +218,7 @@ class cPlayer :
 
 	/** Forces the player to move in the given direction.
 	@deprecated Use SetSpeed instead. */
+	[[deprecated("Use SetSpeed instead.")]]
```

Comment:
That's what I thought. It's just that I got an error because of the lack of this, and I thought maybe we changed.